tests/unit/chambers_of_deputies/test_chamber_of_deputies_dataset.py

```diff
@@ -0,0 +1,203 @@
+            'Flight ticket issue'
+        ]
+
+        present_subquotas = pd.unique(dataset['subquota_description'])
+        for subquota in present_subquotas:
+            assert(subquota in all_subquotas)
+
+    def _parse_float(self, string):
+        return float(string.replace(',', '.'))
+
+    def _read_csv(self, path):
+        return pd.read_csv(path,
+                           encoding='utf-8',
+                           delimiter=";",
+                           quoting=csv.QUOTE_NONE,
+                           dtype={'ideDocumento': np.str,
+                                  'idecadastro': np.str,
+                                  'nuCarteiraParlamentar': np.str,
+                                  'codLegislatura': np.str,
+                                  'txtCNPJCPF': np.str,
+                                  'numRessarcimento': np.str},
+                           converters={'vlrDocumento': self._parse_float,
+                                       'vlrGlosa': self._parse_float,
+                                       'vlrLiquido': self._parse_float,
+                                       'vlrRestituicao': self._parse_float})
+
+    def _read_xz(self, filepath):
+        dtype = {
+            'applicant_id': np.str,
+            'batch_number': np.str,
+            'cnpj_cpf': np.str,
+            'congressperson_document': np.str,
+            'congressperson_id': np.str,
+            'document_id': np.str,
+            'document_number': np.str,
+            'document_type': np.str,
+            'leg_of_the_trip': np.str,
+            'passenger': np.str,
+            'reimbursement_number': np.str,
+            'subquota_group_description': np.str,
+            'subquota_group_id': np.str,
+            'subquota_number': np.str,
+            'term_id': np.str,
+        }
+        return pd.read_csv(filepath, dtype=dtype)
+
+    def _assert_that_the_columns_are_as_expected_before_translation(self, data_frame_2017):
+        expected_columns = [
+            'txNomeParlamentar',
+            'idecadastro',
+            'nuCarteiraParlamentar',
+            'nuLegislatura',
+            'sgUF',
+            'sgPartido',
+            'codLegislatura',
+            'numSubCota',
+            'txtDescricao',
+            'numEspecificacaoSubCota',
+            'txtDescricaoEspecificacao',
+            'txtFornecedor',
+            'txtCNPJCPF',
+            'txtNumero',
+            'indTipoDocumento',
+            'datEmissao',
+            'vlrDocumento',
+            'vlrGlosa',
+            'vlrLiquido',
+            'numMes',
+            'numAno',
+            'numParcela',
+            'txtPassageiro',
+            'txtTrecho',
+            'numLote',
+            'numRessarcimento',
+            'vlrRestituicao',
+            'nuDeputadoId',
+            'ideDocumento'
+        ]
+
+        for column in expected_columns:
+            self.assertIn(column, data_frame_2017.columns)
```

Comment:
This would allow better errors messages (if any):

```python
        for column in expected_columns:
            with self.subTest():
                self.assertIn(column, data_frame_2017.columns)
```